Analysis/R/set_parameters.R

```diff
@@ -396,9 +396,17 @@ for (param in names(taxdat::get_all_config_options())) {
 print("This is the explicit runtime config (printed for debugging).")
 print(config)
 
+# Remove environmental variables to keep clean
+for (param in names(taxdat::get_all_config_options())) {
+  if (param != "stan" & param != "res_time"){
```

Comment:
why do we want to keep res_time?

Reply:
I thought that res_time coming out of the check function was not a string any more, but I was wrong. We can remove this.